src/audio/AudioStream.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function AudioStream( path, listener ) {
```

Comment:
I think I would follow the other classes and keep this as `function AudioStream( listener )`. And add a `setAudioElement( new Audio( path ) )`.